tests/acceptance/features/lib/FilesPageElement/SharingDialog.php

```diff
@@ -103,6 +104,41 @@ private function getExpirationFieldFor($user, $type = 'user') {
 		return $this->find("xpath", \sprintf($this->shareWithExpirationFieldXpath, $user));
 	}
 
+	/**
+	 * click the dropdown button for share actions in the sidebar
+	 *
+	 * @return void
+	 */
+	public function toggleShareActionsDropDown() {
+		$showCrudsBtn = $this->find("xpath", $this->showCrudsXpath);
+		$this->assertElementNotNull(
+			$showCrudsBtn,
+			__METHOD__
+			. " xpath $this->showCrudsXpath could not find show-cruds button"
+		);
+		$showCrudsBtn->click();
+	}
+
+	/**
+	 * open the dropdown for share actions in the sidebar
+	 *
+	 * @return void
+	 */
+	public function openShareActionsDropDown() {
+		$this->toggleShareActionsDropDown();
+		$this->waitTillElementIsNotNull($this->shareWithListDetailsXpath);
+	}
+
+	/**
+	 * open the dropdown for share actions in the sidebar
```

Comment:
```suggestion
	 * close the dropdown for share actions in the sidebar
```
I will fix this after merging, when I am rebasing the PR to which this gets merged.